tox.ini

```diff
@@ -2,19 +2,19 @@
 envlist = py27,py37
 
 [testenv]
-deps = 
+deps =
     pytest
 commands =
     python setup.py install
-    pytest tests
+    pytest {posargs}
```

Comment:
I think we need this for PyPy CIs (here and in the similar lines below):

```suggestion
    pytest {posargs:tests}
```

Reply:
Thanks @Gallaecio, it works